### DISCUSSION TITLE
AI Assistant with Data API builder (and other cool services)

### DISCUSSION THREAD

**Discussion by yorek** [Azure/data-api-builder#2111] [Show and tell]
Build a Retrieval Augmented Generation solution using OpenAI, Azure Functions, Azure Static Web Apps, Azure SQL DB, Data API builder and Text Embeddings. See it in action here: https://ai.microsofthq.vslive.com/ and deploy it in your subscription (and easily change it to work on your data) using AZD! 

![session-assistant-architecture](https://github.com/Azure/data-api-builder/assets/2612362/93ce2f15-675a-4069-95f3-35dabf21d496)

**Comment by jmeeske** [comment]
Your link gives a 404, seems interesting. Can you share some more?

**Comment by yorek** [comment]
There are some DNS issues, in the meantime we get them fixed, I updated the link to https://thankful-hill-032a6de0f.5.azurestaticapps.net that works just fine.

**Comment by grantvine-sc** [comment]
The link doesnt appear to be working again ? Any chance we could get a link to the source of this somewhere ?

**Reply by yorek** [reply]
Due to several updates and also increased security requirements I had to re-deploy everything and I forgot to update this link...so thanks a lot for bringing this up right away. Ue this link: https://ai.microsofthq.vslive.com/. Enjoy! :)
